crates/polars-ops/src/series/ops/is_last.rs

```diff
@@ -0,0 +1,179 @@
+use std::hash::Hash;
+
+use arrow::array::BooleanArray;
+use arrow::bitmap::MutableBitmap;
+use polars_arrow::utils::CustomIterTools;
+use polars_core::prelude::*;
+use polars_core::with_match_physical_integer_polars_type;
+
+pub fn is_last(s: &Series) -> PolarsResult<BooleanChunked> {
+    // fast path.
+    if s.len() == 0 {
+        return Ok(BooleanChunked::full_null(s.name(), 0));
+    } else if s.len() == 1 {
+        return Ok(BooleanChunked::new(s.name(), &[true]));
+    }
+
+    let s = s.to_physical_repr();
+
+    use DataType::*;
+    let out = match s.dtype() {
+        Boolean => {
+            let ca = s.bool().unwrap();
+            is_last_boolean(ca)
+        },
+        Binary => {
+            let ca = s.binary().unwrap();
+            is_last_bin(ca)
+        },
+        Utf8 => {
+            let s = s.cast(&Binary).unwrap();
+            return is_last(&s);
+        },
+        Float32 => {
+            let ca = s.bit_repr_small();
+            is_last_numeric(&ca)
+        },
+        Float64 => {
+            let ca = s.bit_repr_large();
+            is_last_numeric(&ca)
+        },
+        dt if dt.is_numeric() => {
+            with_match_physical_integer_polars_type!(s.dtype(), |$T| {
+                let ca: &ChunkedArray<$T> = s.as_ref().as_ref().as_ref();
+                is_last_numeric(ca)
+            })
+        },
+        #[cfg(feature = "dtype-struct")]
+        Struct(_) => return is_last_struct(&s),
+        #[cfg(feature = "group_by_list")]
+        List(inner) if inner.is_numeric() => {
+            let ca = s.list().unwrap();
+            return is_last_list(ca);
+        },
+        dt => polars_bail!(opq = is_last, dt),
+    };
+    Ok(out)
+}
+
+fn is_last_boolean(ca: &BooleanChunked) -> BooleanChunked {
+    let mut out = MutableBitmap::with_capacity(ca.len());
+    out.extend_constant(ca.len(), false);
+
+    if ca.null_count() == ca.len() {
+        out.set(ca.len() - 1, true);
+    }
+    // TODO supports fast path.
```

Comment:
Here we need to be able to get the reverse slicing of `Bitmap`. I haven't found the corresponding API. Does anyone know about this?